utils/cautils/token_generator.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/smallstep/cli/internal/cryptoutil"
```

Comment:
Format this file with `goimports`, this import should be ordered in the next block of imports.